wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss

```diff
@@ -536,3 +536,33 @@ p {
 hr {
 	height: 1px;
 }
+
+
+/* Details summary block START */
+details {
+	border: 1px solid #e7e7e7;
+	padding: 0 .9em;
+	background: #f7f7f7;
+	margin: 1em 0;
+	display: flex;
+}
+
+details summary::before {
+	content: "\025B8";
+	margin-right: .5em;
+	font-size: 200%;
+	display: inline;
+	vertical-align: bottom;
+	line-height: 1.4;
+}
+
+details[open] summary::before {
+	content: "\025BE";
+}
+
+details summary {
+	display: inline-block;
+	font-weight: bold;
+	cursor: pointer;
+	line-height: 2.8;
+}
```

Comment:
This appears to be for a Sensei thing so maybe move to `_sensei.scss`?

![Screen Shot 2022-10-31 at 2 19 50 PM](https://user-images.githubusercontent.com/1017872/198913088-e6ed2dec-3c4d-40f5-8b78-a8e14dd9d631.jpg)